gravitee/README.md

```diff
@@ -8,8 +8,23 @@ This chart will deploy the following:
 - Gravitee API
 - Gravitee Gateway
 
+## Create a chart archive
+
+To package this chart directory into a chart archive, run:
+
+```
+$ helm package .
+```
+
 ## Installing the Chart
 
+To install the chart using the chart archive, run:
+
+```
+$ helm install gravitee-0.0.1.tgz
```

Comment:
shouldn't be 0.0.2? (see your change in Chart.yaml)

Reply:
It should. I will update.
Thanks for reviewing, @chicco785!